rosetta/object/signature.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package object
+
+import (
+	"github.com/optakt/flow-dps/rosetta/identifier"
+)
+
+type SigningPayload struct {
```

Comment:
For another endpoint a `Signature` type was added. I've put both `SigningPayload` and `Signature` types in a single file called `signature.go`, so that's why currently this deviates from our usual naming pattern for these objects.

I dunno, I could also have them in separate files. Though signatures include the signing payloads, so they're also related.

Reply:
I don't see a reason not to put them in their own file. Also, in the Rosetta model, the field is simply called `Payloads`, so why do you make it `SigningPayload` here? Is there another kind of payload?

It's interesting to see that you call the `unsigned_transaction` just `Transaction` on the struct, but then for the type you feel the need to make it so long? Unless I'm missing a conflict in naming.

Reply:
In the payloads response, the field is called `payloads`, but the type itself is called `SigningPayload` - https://www.rosetta-api.org/docs/models/SigningPayload.html.

I will move it to a separate file though.